iotilebuild/iotile/build/doc/dependencies.py

```diff
@@ -51,7 +51,7 @@ def _get_dependencies(self, component):
         with open(module_settings, "r") as f:
             settings = json.load(f)
 
-        mod = settings['modules'].keys()[0]
+        mod = list(settings['modules'].keys())[0]
```

Comment:
You can actually just call `list(settings['modules'])` here, the `.keys()` is redundant:
http://python-future.org/compatible_idioms.html#dict-keys-values-items-as-a-list